### PR TITLE
[Doc][Kommander]Updated Release Note info for Kommander 1.3.2.

### PR DESCRIPTION
**Pull request by russurquhart1**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-74912
## Description of changes being made
Updated Release Note info for Kommander 1.3.2.

## Checklist
- [ ] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [ ] Create your PR against `main`. 
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

**Review by gracedo — APPROVED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.2/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+| **Default**        | 1.18.10  |
+
+# New features
+
+- Enabled PodSecurityPolicy in Kubecost.
+- Updated license page copy in Kommander UI.
+
+# Bug fixes
+
+- Updated UI to handle new KUDO param types.
```
Comment (cmastr) [inline pages/dkp/kommander/1.3/release-notes/1.3.2/index.md:36]: Do either of these have COPS numbers? Please add.

File: pages/dkp/kommander/1.3/release-notes/1.3.2/index.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# Supported Versions
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**        | 1.17.0  |
+| **Maximum**        | 1.19.x  |
+| **Default**        | 1.18.10  |
+
+# New features
+
+- Enabled PodSecurityPolicy in Kubecost.
```
Comment (cneth) [inline pages/dkp/kommander/1.3/release-notes/1.3.2/index.md:31]: Can we elaborate any on this one?  It's super vague...

**Review by cneth — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/1.3/release-notes/1.3.2/index.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+- Updated license page copy in Kommander UI.
```
[the remark this answers is not in the document]
Comment (gracedo) [reply pages/dkp/kommander/1.3/release-notes/1.3.2/index.md:32]: Looks like that goes with this ticket https://jira.d2iq.com/browse/D2IQ-74321 We can probably just remove it

File: pages/dkp/kommander/1.3/release-notes/1.3.2/index.md (hunk 1 above)
Comment (gracedo) [reply pages/dkp/kommander/1.3/release-notes/1.3.2/index.md:36]: No COPS for these

File: pages/dkp/kommander/1.3/release-notes/1.3.2/index.md (hunk 2 above)
Comment (gracedo) [reply pages/dkp/kommander/1.3/release-notes/1.3.2/index.md:31]: Maybe it's not very informative for users. Can replace with more relevant feat our customers would be more interested in:
```suggestion
- (Kubecost) Move more resources offline for improved air-gapped experience
```

**Pull request merged**